webapi/src/main/scala/org/knora/webapi/responders/v2/resources/CreateResourceV2Handler.scala

```diff
@@ -467,9 +469,9 @@ final case class CreateResourceV2Handler(
                     DateValueInfo(
                       valueHasStartJDN = valueHasStartJDN,
                       valueHasEndJDN = valueHasEndJDN,
-                      valueHasStartPrecision = valueHasStartPrecision,
-                      valueHasEndPrecision = valueHasEndPrecision,
-                      valueHasCalendar = valueHasCalendar,
+                      valueHasStartPrecision = valueHasStartPrecision.toString(),
+                      valueHasEndPrecision = valueHasEndPrecision.toString(),
+                      valueHasCalendar = valueHasCalendar.toString(),
```

Comment:
*Question*: Is there a reason why the `DateValueInfo` should not accept the typed values?

IMHO `DatePrecisionV2` and `CalendarNameV2` are perfectly good types and should be turned into a `String` when building the query.

Reply:
also done in https://github.com/dasch-swiss/dsp-api/pull/3276/commits/550c7b20ad7a0c78231ea9605491bca3fcebdc86 - let me know what you think

Reply:
> Do you think that is an issue, then I would add a new model representing it; otherwise I will just reuse them.

Not in this case. Theses data structures are simple and do not depend on something weird. In the long run we would of course want to move them into the slice.

Reply:
makes sense, thanks!